samples/subsys/fs/littlefs/sample.yaml

```diff
@@ -0,0 +1,7 @@
+sample:
+  name: littlefs filesystem sample
+tests:
+  sample.subsys.fs.littlefs:
+    build_only: true
+    platform_whitelist: particle_xenon
```

Comment:
Readme mentions 2 tested platforms `nrf52840_pca10056 ` and `particle_xenon` and here only one of them is whitelisted, any reason for that?

Reply:
I think PCA10056 was added later just because it's more common.  I can put both there.